pkg/controller/plan/kubevirt.go

```diff
@@ -658,22 +658,22 @@ func (r *KubeVirt) getPVCs(vmRef ref.Ref) (pvcs []*core.PersistentVolumeClaim, e
 	err = r.Destination.Client.List(
 		context.TODO(),
 		pvcsList,
-		r.getListOptionsNamespaced(),
+		&client.ListOptions{
+			LabelSelector: labels.SelectorFromSet(map[string]string{
+				"migration": string(r.Migration.UID),
+				kVM:         vmRef.ID,
+			}),
+		},
 	)
-
 	if err != nil {
 		err = liberr.Wrap(err)
-		return
```

Comment:
I'd still suggest to return here even if the code below wouldn't do anything but return - as it's more explicit  and consistent with the rest of the code